Composer/packages/client/src/CreationFlow/LocationBrowser/LocationSelectContent.tsx

```diff
@@ -2,60 +2,19 @@
 // Licensed under the MIT License.
 
 /** @jsx jsx */
-import path from 'path';
-
 import { jsx } from '@emotion/core';
-import { Fragment, useEffect, useState, useContext, useRef } from 'react';
+import { Fragment, useContext, useRef } from 'react';
 
 import { FileSelector } from './FileSelector';
 import { StoreContext } from './../../store';
 import { FileTypes } from './../../constants';
 
 export function LocationSelectContent(props) {
```

Comment:
You can write it like this to simplify the code.

```suggestion
export function LocationSelectContent({ onOpen, allowOpeningBot = true, focusedStorageFolder, updateCurrentPath, currentPath }) {
```